README.md

```diff
@@ -53,7 +53,7 @@ Resources
 -   Ask and answer questions in our [Frequently Asked Questions](FAQs.md) page.
 -   If you'd like to contact us directly, drop us a mail at cortx-questions@seagate.com.
 -   We have _CORTX stickers_ available to anyone who would like one! Request yours by filling out [this form](https://forms.office.com/Pages/ResponsePage.aspx?id=aiFm1EPGSkOcLgV0SMF8vgKFhRIEtppAkBdqKlO8waVUMVk1Uk9PWjc2NVhNTUU1WkdVTTFIVlNMSS4u)
--   Subscribe to the our [developer newsletter](https://www.seagate.com/promos/developer-newsletter/) to and stay up to date on the latest CORTX developments, news, and events.
```

Comment:
Is this necessary Saumya?  Both links work for me.

Reply:
It did not work for me and Gaurav today while testing @johnbent, hence I changed it